File: ABASim-SPA/src/app/_models/inboxMessage.ts
export interface InboxMessage {
    id: number;
    senderId: number;
    senderName: string;
    senderTeam: string;
    receiverId: number;
    receiverName: string;
    receiverTeam: string;
    subject: string;
    body: string;
    messageDate: string;
    isNew: number;
}

File: ABASim.api/Models/InboxMessage.cs
namespace ABASim.api.Models
{
    public class InboxMessage
    {
        public int Id { get; set; }

        public int SenderId { get; set; }

        public string SenderName { get; set; }

        public string SenderTeam { get; set; }

        public int ReceiverId { get; set; }

        public string ReceiverName { get; set; }

        public string ReceiverTeam { get; set; }

        public string Subject { get; set; }

        public string Body { get; set; }

        public string MessageDate { get; set; }

        public int IsNew { get; set; }
    }
}